relayer-cli/tests/fixtures/two_chains.toml

```diff
@@ -3,12 +3,13 @@ timeout = '10s'
 strategy = 'naive'
 
 [[chains]]
-id = 'ibc0'
+id = 'ibc-0'
```

Comment:
Is this change needed because the identifiers of the chains spawned by the Go relayer setup scripts have changed?

Reply:
Yes, they changed

Reply:
Hmm, i don't understand why this needs change. We are still on stargate-4. This change is coming in stargate-5.

Reply:
do you want me to change this back to `ibc0` ?